### PR TITLE
Add fastapi to `pip install mlem[streamlit]`

### PR DESCRIPTION
**Pull request by aguschin**
@mike0sv, IIUC it should work like this. Could you please check?

### CODE REVIEW

**Comment by mike0sv** [comment]
I initially did it like this, but decided to make those optional for streamlit. If you check streamlit server get_requirements, fastapi reqs are added if you dont have external backend address.

We can make them required though, but you also need to fix `ext.py` I guess

**Comment by aguschin** [comment]
Got your idea. IMO we still better install `fastapi` with `pip install mlem[streamlit]`, since otherwise >90% of users will encounter errors and have to do it manually. Even if it's not always necessary requirement.

File: setup.py [hunk 1]
```diff
@@ -70,6 +70,7 @@
     "xgboost": ["xgboost"],
     "lightgbm": ["lightgbm"],
     "fastapi": ["uvicorn", "fastapi"],
+    "streamlit": ["uvicorn", "fastapi", "streamlit", "streamlit_pydantic"],
     "sagemaker": ["boto3", "sagemaker"],
```
Comment (aguschin) [inline setup.py:74]: maybe:
```suggestion
    "sagemaker": ["docker", "boto3", "sagemaker"],
```
? @mike0sv 
or you had some idea behind this as well?

Comment (mike0sv) [reply setup.py:74]: probably not :)

**Review by mike0sv — APPROVED** [review]

**Pull request merged**